packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -216,6 +224,24 @@ class TimeInput extends React.Component {
     });
   }
 
+  /* When Time-Input is used with date-time-picker time-input In missing hour scenario Time-Input skips state update due to
+     prevProps and props comparsion. To handle this scenario propValue is being validated with prevState values in below method
```

Comment:
- "Time-Input" -> "time-input"
- "In" -> "in"
- "comparision" -> "comparison"
- "propValue" -> " the value prop"